src/renderer/prefComponents/common/textBox/index.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <section class="pref-text-box-item" :class="{'ag-underdevelop': disable}">
+    <div class="description">
+      <span>{{description}}</span>
+      <i class="el-icon-info" v-if="more"
+        @click="handleMoreClick"
+      ></i>
+    </div>
+    <el-input
+      class="input"
+      :class="{error: invalidInput}"
+      :placeholder="defaultValue"
+      v-model="inputText"
+      @change="handleInputChange"
```

Comment:
It's better to handle `input` event? Because `change` event only fired when the input loose focus. we can listen the `input` event and add a throttle.

Reply:
I think the background color of input is too grey, like it's a disabled input. it's also too width, I think 180px is OK, the same as drop down component.

Reply:
Yeah, I changed the width but the colors are used from Element UI, so we should change this in another PR. I don't know what to change there, also el-input has no `input` event according documents?

Reply:
I just ask the author(we are in the same company) of Element, He said el-input also support native input element event, like `input`. So you can use `@input="handlerInput"`.

Reply:
Yeh, Element didn't mention it in their document.